test/orbit/authentication/user_test.exs

```diff
@@ -43,4 +45,50 @@ defmodule Orbit.Authentication.UserTest do
                |> Repo.update()
     end
   end
+
+  describe "get_names_from_employee/1" do
+    test "looks up name in Employee" do
+      Repo.insert!(%Employee{
+        first_name: "Arthur",
+        preferred_first: "Art",
+        last_name: "Read",
+        email: "arthur@mbta.com",
+        badge_number: "123456789"
+      })
+
+      user =
+        Repo.insert!(%User{
+          email: "arthur@mbta.com",
+          permissions: [:operator_sign_in]
+        })
```

Comment:
**suggestion:** these could both be accomplished a bit more idiomatically with the `insert!` macro from ExMachina and our existing factories.

Reply:
👍 7f75eed